tests/unit/datarobot_drum/drum/language_predictors/test_base_language_predictor.py

```diff
@@ -113,11 +113,52 @@ def chat_hook(completion_request):
             ANY,
             ["How are you"],
             association_ids=ANY,
-            skip_drift_tracking=True,
-            skip_accuracy_tracking=True,
         )
         # Compare features dataframe separately as this doesn't play nice with assert_called
-        assert mock_mlops.report_predictions_data.call_args.args[0]["prompt"].values[0] == "Hello!"
+        assert (
+            mock_mlops.report_predictions_data.call_args.args[0]["promptText"].values[0] == "Hello!"
+        )
+
+    def test_prompt_column_name(self, chat_python_model_adapter, mock_mlops):
```

Comment:
Can we have a test for default prompt column name?

Reply:
[test_chat_with_mlops](https://github.com/datarobot/datarobot-user-models/blob/bacfedd9ac76b189df43b8e2f55ce74a7a3d1078/tests/unit/datarobot_drum/drum/language_predictors/test_base_language_predictor.py#L84) is testing this implicitly